src/debug/telemetry/Kconfig

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+config TELEMETRY
```

```suggestion
config SOF_TELEMETRY
```
... otherwise high risk of clashing with Zephyr or some Zephyr dependency.

https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-3-macro-name-collisions

This is not theoretical : we have already experienced name clashes like this.